cypress/e2e/tests/migration/dynamic-report/issues/filter.sorting.pagination.test.ts

```diff
@@ -59,7 +57,7 @@ describe(["@tier3"], "Filtering, sorting and pagination in Issues", function ()
         archetype.create();
         cy.fixture("application").then((appData) => {
             cy.fixture("analysis").then((analysisData) => {
-                for (let i = 0; i < 6; i++) {
+                for (let i = 0; i < 1; i++) {
```

Comment:
A `for` loop from i = 0 while i<1 is just 0, so the `for` is not needed

Reply:
Forgot to revert back to 6, so your comment is not relevant any more